pkg/controller/syncsetinstance/syncsetinstance_controller_test.go

```diff
@@ -1539,3 +1539,16 @@ func TestSSIReApplyDuration(t *testing.T) {
 		}
 	}
 }
+
+func TestFilterApplyError(t *testing.T) {
+	err1 := "error when creating \"/tmp/apply-475927931\": namespaces \"openshift-am-config\" not found"
+	expectedFilteredErr1 := "namespaces \"openshift-am-config\" not found"
+	if filterApplyError(err1) != expectedFilteredErr1 {
+		t.Fatalf("expected temporary file to be trimmed from error message")
+	}
+
+	err2 := "unable to parse \"[ { \\\"op\\\": \\\"replace\\\", \\\"path\\\": \\\"/data/foo\\\", \\\"value\\\": \\\"baz-json\\\" ]\": yaml: did not find expected ',' or '}'"
```

Comment:
You can save yourself a bunch of escapes here by using \`unable to parse ...\` instead of "unable to parse ...".

Reply:
Shortened. This was just an error that we generate from syncset I use as a failing patch test.